integration-cli/docker_cli_plugins_test.go

```diff
@@ -37,6 +49,11 @@ func (s *DockerSuite) TestPluginBasicOps(c *check.C) {
 	out, _, err = dockerCmdWithError("plugin", "remove", pNameWithTag)
 	c.Assert(err, checker.IsNil)
 	c.Assert(out, checker.Contains, pNameWithTag)
+
+	_, err = os.Stat(filepath.Join(dockerBasePath, "plugins", string(id)))
+	if !os.IsNotExist(err) {
```

Comment:
should we check if `err` is `nil` here as well? the "fatal" may not be very useful if it's `nil`